.changeset/bright-signals-share.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/playground-ui': minor
+---
+
+Export the Trace Intelligence experience with injectable request and navigation adapters so product hosts can render it outside OSS Studio.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)